apps/timetable/admin/ui/css/admin.css

```diff
@@ -411,6 +411,10 @@
     margin-top: -10px;
 }
 
+.gh-event-delete button {
+    width: 100%;
```

Comment:
The colour of the button might need to change to black to be in line with the admin UI design